Packs/CommonTypes/IndicatorTypes/reputation-email.json

```diff
@@ -1,75 +1,85 @@
 {
-  "id": "emailRep",
-  "version": -1,
-  "fromVersion": "5.0.0",
-  "modified": "2019-07-18T08:56:47.982384118Z",
-  "sortValues": null,
-  "commitMessage": "",
-  "shouldPublish": false,
-  "shouldCommit": false,
-  "regex": "\\b[A-Za-z0-9.!#$%&'*+/=?^_`{|}~\\p{L}-]{1,64}(@|\\[@\\])[A-Za-z0-9\\p{L}.-]{1,255}\\.[A-Za-z]{2,}\\b",
-  "details": "Email",
-  "prevDetails": "Email",
-  "reputationScriptName": "",
-  "reputationCommand": "email",
-  "enhancementScriptNames": [
-    "EmailReputation"
-  ],
-  "system": true,
-  "locked": false,
-  "disabled": false,
-  "file": false,
-  "updateAfter": 0,
-  "mergeContext": false,
-  "formatScript": "ExtractEmailV2",
-  "contextPath": "Account.Email(val.Address \u0026\u0026 val.Address === obj.Address)",
-  "contextValue": "Address",
-  "excludedBrands": [],
-  "defaultMapping": {
-    "blocked": {
-      "simple": "",
-      "complex": {
-        "root": "Account",
-        "filters": [],
-        "accessor": "Blocked",
-        "transformers": [
-          {
-            "operator": "uniq",
-            "args": {}
-          }
-        ]
-      }
+    "id": "emailRep",
+    "version": -1,
+    "fromVersion": "5.0.0",
+    "modified": "2019-07-18T08:56:47.982384118Z",
+    "shouldPublish": false,
+    "shouldCommit": false,
+    "regex": "\\b[A-Za-z0-9.!#$%\u0026'*+/=?^_`{|}~\\p{L}-]{1,64}(@|\\[@\\])[A-Za-z0-9\\p{L}.-]{1,255}\\.[A-Za-z]{2,}\\b",
+    "details": "Email",
```

Comment:
The regex changed as well, planned? reason?

Reply:
It only encodes `&` which is later decoded:
![image](https://user-images.githubusercontent.com/24267931/158193923-a77c9cfe-d309-4afd-a3b8-29fcb9aec1ea.png)